cmake/Scripts/configure

```diff
@@ -48,6 +48,7 @@ Optional Packages:
   --with-superlu=PATH     user defined path to SuperLU library
   --with-umfpack=PATH     use UMFPACK/SuiteSparse from a specified location
   --with-ert=PATH         Use ERT libraries
+  --with-zoltan=PATH      Use ZOLTAN libraries
```

Comment:
By the way, there are some questionable constructs in the script itself.  For instance, using
```sh
test -n "${OPTARG#with-${pkgname}}" && pkgloc=${OPTARG#*=} || pkgloc=""
```
as quick way of making `pkgloc` either empty or a (hopefully reasonable) directory string is not guaranteed to work due bindings of shell logical operators.

I'll run the configure script to the analysis tool at [ShellCheck.net](http://shellcheck.net).